entity-framework/core/get-started/uwp/getting-started.md

```diff
@@ -13,49 +13,42 @@ uid: core/get-started/uwp/getting-started
 
 # Getting Started with EF Core on Universal Windows Platform (UWP) with a New Database
 
-> [!NOTE]  
-> Temporarily this tutorial uses EF Core 1.1. UWP has not been updated yet to support .NET Standard 2.0 which is required for compatibility with EF Core 2.0. Once it is, we will update the tutorial to use the new version. 
+-> [!NOTE]
+-> This tutorial uses EF Core 2.0.1. EF Core 2.0.0 lacks some crucial bugfixes required for a good UWP experience.
```

Comment:
If we're going to mention 2.0.1, we should probably also mention that it will be released as the 2.0.3 meta package, or at least add an appropriate link when available.

Reply:
Do you think UWP customers will use the ASP.NET Core metapackage useful?

Reply:
Good point. 😸 However, the announcements/tweets/etc. people will see will likely say 2.0.3, so I think it's still worth saying something like, "released as part of the wider 2.0.3 ASP.NET/EF Core patch release" or something like that.

Reply:
Ok, with mentioning it between parenthesis.